sdcm/logcollector.py

```
@@ -1585,6 +1585,13 @@ class SSLConfCollector(BaseSCTLogCollector):
     cluster_log_type = "ssl-conf"
 
 
+class DiagnosticsLogCollector(BaseSCTLogCollector):
+    log_entities = [
+        DirLog(name="diagnostics/*", search_locally=True),
+    ]
```

Ruff (0.15.18)

[warning] Mutable default value for class attribute

(RUF012)